src/controllers/login-controller.js

```diff
@@ -0,0 +1,35 @@
+const loginService = require('../services/login-service');
+
+const getData = async (req, res) => {
+    const { body } = req;
+    if(
+        !body.mail ||
+        !body.password
+    ){
+        res.send({
+            status: "FAILED",
+            data: { error: "Missing keys" }
+        });
+        return;
+    }
```

Comment:
this validations are ok, but we can try layer up those validations, check this package for be able to do so https://express-validator.github.io/docs/

if you have any questions let me know

Reply:
or not necessarily in a upper layer using this package  https://joi.dev/api/?v=17.7.0

i will leave it up to you